hq/test/schedule/IamAuditTest.scala

```diff
@@ -2,12 +2,10 @@ package schedule
 
 import com.gu.anghammarad.models.Stack
 import model.{UserWithOutdatedKeys, _}
-import com.gu.anghammarad.models.{Email, Notification, Preferred, AwsAccount => AwsAccountTarget}
-import model._
 import org.joda.time.DateTime
 import org.scalatest.{FreeSpec, Matchers}
 import schedule.IamAudit._
-import utils.attempt.FailedAttempt
+import schedule.IamMessages.createMessage
 
 class IamAuditTest extends FreeSpec with Matchers {
```

Comment:
I've changed the tests which were testing `makeCredentialsNotification` to now test the `createMessage` function, because this was the main part of the feature that was not covered by other tests.